protocol/contracts/v2/TempleDebtToken.sol

```diff
@@ -292,11 +298,18 @@ contract TempleDebtToken is ITempleDebtToken, TempleElevatedAccess {
                 _totalBaseShares,
                 _debtorPrincipal
             );
-            _burnAmount -= _baseDebtRepaid;
+            
+            unchecked {
+                _burnAmount -= _baseDebtRepaid;
+            }
 
             // The minimum of what debt is still outstanding, and the requested amount to be burned
             _debtorDebtRepaid = _debtorCheckpoint < _burnAmount ? _debtorCheckpoint : _burnAmount;
             _burnAmount -= _debtorDebtRepaid;
```

Comment:
There was a bug introduced here (duplicated the `_debtorDebtRepaid` being subtracted), which was fixed later in https://github.com/TempleDAO/temple/pull/842/commits/a657348ea1abdc9e8a9393dd1857e034ac2507b0